### PR TITLE
Neuromag 122 support

### PR DESCRIPTION
**Pull request by agramfort**
this adds support for Neuromag 122 support. System in Heidelberg

cc @adykstra @Diptyajit7

### CODE REVIEW

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -39,7 +38,7 @@ def _prepare_topo_plot(inst, ch_type, layout):
 
     if layout is None and ch_type is not 'eeg':
         from ..channels import find_layout
-        layout = find_layout(info)
+        layout = find_layout(info)  # XXX : why not passing ch_type???
```
[the remark this answers is not in the document]
Comment (larsoner) [reply mne/viz/topomap.py]: I always get lost in this code, I'm often not sure what the logic is :(

**Comment by agramfort** [comment]
Green

**Comment by larsoner** [comment]
Thanks @agramfort

**Comment by dasdiptyajit** [comment]
Great! @agramfort

**Pull request merged**